packages/react/src/tooltip-content/TooltipContent.tsx

```diff
@@ -0,0 +1,65 @@
+import { useComposedRefs } from "@radix-ui/react-compose-refs";
+import * as RadixTooltip from "@radix-ui/react-tooltip";
+import { forwardRef, useRef } from "react";
+
+import { AnimatePresence } from "../animate-presence";
+import { Box, type BoxProps } from "../box";
+import { Text } from "../text";
+import { theme } from "../theme";
+import { useTooltipContext } from "../tooltip-context";
+import { Transition } from "../transition";
+import * as styles from "./TooltipContent.css";
+
+type TooltipContentProps = BoxProps<typeof RadixTooltip.Content>;
+
+export const TooltipContent = forwardRef<HTMLDivElement, TooltipContentProps>(
+  (
+    { align = "center", children, className, sideOffset = 2, ...props },
```

Comment:
let's move the z=popover to here
```suggestion
    { align = "center", children, className, sideOffset = 2, z = "popover", ...props },
```